sound/soc/codecs/rt5682.h

```diff
@@ -177,7 +177,7 @@
 #define RT5682_TEST_MODE_CTRL_4			0x0148
 #define RT5682_TEST_MODE_CTRL_5			0x0149
 #define RT5682_PLL1_INTERNAL			0x0150
-#define RT5682_PLL2_INTERNAL			0x0151
+#define RT5682_PLL2_INTERNAL			0x0156
```

Comment:
different patch?

Reply:
The PLL2 is used for SoundWire interface, and the PLL2 register address was wrong.